lavaplay/ws.py

```diff
@@ -38,22 +36,18 @@ def __init__(
         self._headers = {
             "Authorization": password,
             "User-Id": str(user_id),
-            "Client-Name": f"Lavaplay.py/{__version__}",
-            "Num-Shards": str(shards_count)
```

Comment:
Should keep this

Reply:
Okay, I do, but why exactly?

Reply:
https://github.com/HazemMeqdad/lavaplay.py/pull/49/commits/36b4386b25405997082b3987e9a76db884c20c7c Good

Reply:
Because shartds still used